criipto/criipto-verify-express
# Initial implementation

To test locally:

```
npm install
npm run build
npm link
cd example
npm install
npm link @criipto/verify-express
npm start
```
